go/oasis-test-runner/oasis/network.go

```diff
@@ -638,8 +644,16 @@ func (net *Network) startOasisNode(
 
 	cfg.Common.DataDir = node.DataDir()
 	cfg.Common.Log.Level = make(map[string]string)
-	cfg.Common.Log.Level["default"] = "debug"
-	cfg.Common.Log.Format = "json"
+	if net.Config().NodeLogLevel != "" {
+		cfg.Common.Log.Level["default"] = net.Config().NodeLogLevel
+	} else {
+		cfg.Common.Log.Level["default"] = "debug"
```

Comment:
Should this be moved to `config.DefaultConfig()`?

Reply:
Probably not, since setting the default log level to `debug` is not a sensible default and is specific to the test runner.  If the fixture was loaded from a file, the log level and format fields might not be populated, so we still need these checks here.